gulpfile.js

```diff
@@ -232,11 +241,11 @@ gulp.task('s3', function (callback) {
       console.log('The version already exists in S3, returning gracefully...');
       return callback();
     }
-    
+
     gulp.src('./styleguide/**/*')
-      .pipe(s3(awsConfig))
-      .on('end', callback);
-    });
+    .pipe(s3(awsConfig))
+    .on('end', callback);
+  });
```

Comment:
I think the lines starting with dot should be intended as they were (two spaces more than the `gulp.src` line